specs/tables.md

```diff
@@ -36,53 +36,65 @@ Proved by the state circuit.
-| $counter | $isWrite  | CallContext                | $callID  | IsRoot                     | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | IsCreate                   | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | CodeSource                 | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | ProgramCounter             | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | StackPointer               | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | GasLeft                    | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | MemorySize                 | $value      | 0           | 0           |
-| $counter | $isWrite  | CallContext                | $callID  | StateWriteCounter          | $value      | 0           | 0           |
-|          |           |                            |          |                            |             |             |             |
-| $counter | $isWrite  | Stack                      | $callID  | $stackPointer              | $value      | 0           | 0           |
-| $counter | $isWrite  | Memory                     | $callID  | $address                   | $value      | 0           | 0           |
-| $counter | $isWrite  | AccountStorage             | $address | $key                       | $value      | $valuePrev  | 0           |
+Type sizes:
+
+> - **txID**: 24 bytes
+> - **address**: 160 bytes
+> - **callID**: 25 bytes
+> - **stackPointer**: 10 bytes
+> - **memoryAddress**: 40 bytes
+> - **Memory -> value, valuePrev**: 1 byte
+> - **storageKey**: field size, RLC encoded (Random Linear Combination)
+> - **value, valuePrev**: variable size, depending on *Key 1 (Tag)* and *Key 3*
+>   where appropiate.
+
+| 0 *rwc*  | 1 *isWrite* | 2 *Key 1 (Tag)*            | 3 *Key 2* | 4 *Key 3*                  | 5 *Key 4*   | 6 *Value 1* | 7 *Value 2* | 8 *Aux 1* | 9 *Aux 2*       |
+| ---      | ---         | ---                        | ---       | ---                        | ---         | ---         | ---         | ---       | ---             |
+|          |             | *RwTableTag*               |           |                            |             |             |             |           |                 |
+| $counter | true        | TxAccessListAccount        | $txID     | $address                   |             | $value      | $valuePrev  | 0         | 0               |
+| $counter | true        | TxAccessListAccountStorage | $txID     | $address                   | $storageKey | $value      | $valuePrev  |           | 0               |
```

Comment:
```
| $counter | $isWrite    | TxAccessListAccount        | $txID     | $address                   |             | $value      | $valuePrev  | 0         | 0               |
| $counter | $isWrite    | TxAccessListAccountStorage | $txID     | $address                   | $storageKey | $value      | $valuePrev  |           | 0               |
```

Reply:
From a discussion with @han0110, for AccessList the operations are always WRITE.  Why do you suggest it's variable?

Reply:
I don't think the operations for AccessList are always WRITE.

To calculate gas for `sload` and `sstore`, which depends on whether a storage has been accessed before (whether it's warm), we need to read AccessList.

Reply:
For access list, the write operation will contain the previous value, so I think we don't need a read here if we always try to write it to true.

Reply:
I see what you mean. I may need to update #91 accordingly

Reply:
will still be great if adding a missing `0` there